crucible-mir-comp/src/Mir/Compositional/Override.hs

```diff
@@ -90,7 +89,18 @@ printSpec ::
     OverrideSim (p sym) sym MIR rtp args ret (RegValue sym MirSlice)
 printSpec ms = do
     let str = show $ MS.ppMethodSpec (ms ^. msSpec)
-    let bytes = Text.encodeUtf8 $ Text.pack str
+    let str2 = unlines
+          [ str
+          , "pre allocs = " ++ show (ms ^. msSpec . MS.csPreState . MS.csAllocs)
+          , "pre pointsto = " ++ show (ms ^. msSpec . MS.csPreState . MS.csPointsTos)
+          , "pre conds = " ++ show (ms ^. msSpec . MS.csPreState . MS.csConditions)
+          , "pre vars = " ++ show (ms ^. msSpec . MS.csPreState . MS.csFreshVars)
+          , "post allocs = " ++ show (ms ^. msSpec . MS.csPostState . MS.csAllocs)
+          , "post pointsto = " ++ show (ms ^. msSpec . MS.csPostState . MS.csPointsTos)
+          , "post conds = " ++ show (ms ^. msSpec . MS.csPostState . MS.csConditions)
+          , "post vars = " ++ show (ms ^. msSpec . MS.csPostState . MS.csFreshVars)
```

Comment:
It might be worth deduplicating the `ms ^. msSpec . MS.csPreState` and `ms ^. msSpec . MS.csPostState` subexpressions here, as they're repeated a lot.